tests/ops/integration_tests/saas/test_aircall_task.py

```diff
@@ -0,0 +1,124 @@
+from tests.ops.graph.graph_test_util import assert_rows_match
+
+CONFIG = get_config()
+
+
+@pytest.mark.integration_saas
+@pytest.mark.integration_aircall
+def test_aircall_connection_test(aircall_connection_config) -> None:
+    get_connector(aircall_connection_config).test_connection()
+
+@pytest.mark.integration_saas
+@pytest.mark.integration_aircall
+@pytest.mark.asyncio
+async def test_aircall_access_request_task_by_email(
+    db,
+    policy,
+    aircall_connection_config,
+    aircall_dataset_config,
+    aircall_identity_email,
+) -> None:
+    """Full access request based on the aircall SaaS config"""
+
+    privacy_request = PrivacyRequest(
+        id=f"test_aircall_access_request_task_{random.randint(0, 1000)}"
+    )
+    identity = Identity(**{"email": aircall_identity_email})
+    privacy_request.cache_identity(identity)
+
+    dataset_name = aircall_connection_config.get_saas_config().fides_key
+    merged_graph = aircall_dataset_config.get_graph()
+    graph = DatasetGraph(merged_graph)
+
+    v = await graph_task.run_access_request(
+        privacy_request,
+        policy,
+        graph,
+        [aircall_connection_config],
+        {"email": aircall_identity_email},
+        db,
+    )
+
+    assert_rows_match(
+        v[f"{dataset_name}:customer"],
+        min_size=1,
+        keys=[
+            "id",
+            "direct_link",
+            "name",
+            "email",
+            "available",
+            "availability_status",
+            "created_at",
+            "time_zone",
+            "language",
+            "wrap_up_time",
+            "numbers",
+        ],
+    )
+
+    assert_rows_match(
+        v[f"{dataset_name}:contact"],
+        min_size=1,
+        keys=[
+            "id",
+            "first_name",
+            "last_name",
+            "company_name",
+            "information",
+            "is_shared",
+            "direct_link",
+            "created_at",
+            "updated_at",
+            "phone_numbers",
+            "emails",
+        ],
+    )
+
+    assert_rows_match(
+        v[f"{dataset_name}:calls"],
+        min_size=1,
+        keys=[
+            "id",
+            "direct_link",
+            "direction",
+            "status",
+            "missed_call_reason",
+            "started_at",
+            "answered_at",
+            "ended_at",
+            "duration",
+            "voicemail",
+            "recording",
+            "asset",
+            "raw_digits",
+            "user",
+            "contact",
+            "archived",
+            "assigned_to",
+            "tags",
+            "transferred_by",
+            "transferred_to",
+            "teams",
+            "number",
+            "cost",
+            "country_code_a2",
+            "pricing_type",
+            "comments"
+        ],
+    )
+    # verify we only returned data for our identity email
+    assert v[f"{dataset_name}:contact"][0]["emails"][0]["value"] == aircall_identity_email
```

Comment:
add erasure tests (and fixtures) for contacts and calls

Reply:
Erasure request added.but,there is no api for create calls.Only possible to make a  call via Aircall App.So,now only checked contact deletion.